R/constructor.R

```diff
@@ -91,3 +109,14 @@ as_names <- function(x, named = FALSE) {
   }
   lapply(x, as.name)
 }
+
+has_S7_symbols <- function(env, ...) {
```

Comment:
Maybe `has_S7_imports`?

Reply:
That's fine, but slightly less accurate. It also needs to return `TRUE` for the S7 namespace itself, which does not "import" S7 symbols.

Reply:
Hmmm, true.